cmd/brew-portable-package.rb

```diff
@@ -20,7 +20,7 @@
   safe_system "brew", "test", name
   puts "Library linkage:"
   if OS.linux?
-    puts Utils.open_read "ldd", "#{f.bin}/#{name.gsub(/^portable-/, "")}"
+    puts Utils.popen_read "ldd", "#{f.bin}/#{name.gsub(/^portable-/, "")}"
```

Comment:
This doesn't assume Linuxbrew, though, but just Homebrew/brew's Linux support, no?. I'd rather avoid relying on commands like that until they are in upstream Homebrew.